dashboard/app/views/layouts/application.html.haml

```diff
@@ -9,9 +9,10 @@
     /[if IE 9]
       %meta{'http-equiv' => 'X-UA-Compatible', content: 'IE=Edge'}
     = stylesheet_link_tag    'application', media: 'all'
-    = javascript_include_tag 'application'
     = javascript_include_tag "js/#{js_locale}/common_locale"
     %script{src: minifiable_asset_path('js/essential.js')}
+    %script{src: minifiable_asset_path('js/layouts/application.js')}
+    = javascript_include_tag 'application'
```

Comment:
This pair of lines strikes me as cryptic and probably merits a comment - especially since I expect ordering is important.